internal/controller/paasns_controller.go

```diff
@@ -49,37 +49,39 @@ func (pr PaasNSReconciler) GetScheme() *runtime.Scheme {
 //
 
 func (r *PaasNSReconciler) GetPaasNs(ctx context.Context, req ctrl.Request) (paasns *v1alpha1.PaasNS, err error) {
-	paasns = &v1alpha1.PaasNS{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: req.Name,
-		},
-	}
+	paasns = &v1alpha1.PaasNS{}
 	logger := getLogger(ctx, paasns, paasns.Kind, req.Name)
-	logger.Info("Reconciling the PaasNs object")
+	logger.Info("Reconciling PaasNs")
 
 	if err = r.Get(ctx, req.NamespacedName, paasns); err != nil {
-		if errors.IsNotFound(err) {
-			// Something fishy is going on
-			// Maybe someone cleaned the finalizers and then removed the PaasNs resource?
-			logger.Info(req.NamespacedName.Name + " is already gone")
-			// return ctrl.Result{}, fmt.Errorf("PaasNs object %s already gone", req.NamespacedName)
-			return nil, nil
+		return nil, client.IgnoreNotFound(err)
```

Comment:
Are we sure we don't want to know (informational) that there was an apparent inconsistency between configured state and reality?

Reply:
Nah, this happens after the object has been finalized and thus removed by k8s. The reconcile after that, falls into the catagory of a reconciliation for an object which cannot be find. There can't be another case. I'm personally not interested in logging those requests.

Reply:
OK